bubble.rb

```diff
@@ -0,0 +1,67 @@
+def bubble_sort(array)
+
+count = array.length
+x = 0
+i = 0
+
+while x < count - 1
+
+while i < count - x - 1
+if array[i] > array[i + 1]
+leftNumb = array[i]
+rightNumb = array[i + 1]
+array[i] = rightNumb
+array[i + 1] = leftNumb
+end
+
+i = i + 1
+end
+
+x = x + 1
+end
```

Comment:
**[REQUIRED]** 
* Sorts an array of numbers in increasing order.
![Screenshot from 2019-12-19 12-19-53](https://user-images.githubusercontent.com/44072711/71194628-4fd54780-225a-11ea-8467-6fdc85e0b30f.png)